test/evergreen.yml

```diff
@@ -4000,24 +3993,6 @@ buildvariants:
   batchtime: 480 # 3 times a day
   run_on:
   - ubuntu2004-test
-  expansions:
```

Comment:
I think you need these because in 6.0 we don't have the same defaults for the cmake process.